packages/@ember/application/globals-resolver.js

```diff
@@ -7,10 +7,34 @@ import { get, findNamespace } from '@ember/-internals/metal';
 import { assert, info } from '@ember/debug';
 import { capitalize, classify, dasherize, decamelize } from '@ember/string';
 import { Object as EmberObject } from '@ember/-internals/runtime';
-import validateType from './lib/validate-type';
 import { getTemplate } from '@ember/-internals/glimmer';
 import { DEBUG } from '@glimmer/env';
 
+function validateType(resolvedType, parsedName) {
```

Comment:
This seems to only be used in the one location below, can we remove this function and inline the implementation down there instead? (allows us to strip out the function as well)

Reply:
yeah I was going to do that at first, but then wanted to confirm if that is fine